plugins/catalog-backend/src/tests/migrations.test.ts

```diff
@@ -308,4 +308,28 @@ describe('migrations', () => {
       await knex.destroy();
     },
   );
+
+  it.each(databases.eachSupportedId())(
+    '20241003170511_alter_target_in_locations.js, %p',
+    async databaseId => {
+      const knex = await databases.init(databaseId);
+
+      await migrateUntilBefore(
+        knex,
+        '20241003170511_alter_target_in_locations.js',
+      );
+
+      await migrateUpOnce(knex);
+      const columnInfo = await knex('locations').columnInfo();
+      expect(columnInfo.target.type).toBe('text');
+
+      await migrateDownOnce(knex);
+      const revertedColumnInfo = await knex('locations').columnInfo();
+      expect(revertedColumnInfo.target.type).toMatch(
+        /^(varchar|character varying)$/,
+      );
```

Comment:
Please try to insert and read data instead, rather than just checking the schema